security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -307,7 +312,7 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 			secret, err := s.st.GenerateSecret(ctx, conID, resourceName, token)
 			if err != nil {
 				sdsServiceLog.Errorf("%s Close connection. Failed to get secret for proxy %q from "+
-					"secret cache: %v", conIDresourceNamePrefix, discReq.Node.Id, err)
+						"secret cache: %v", conIDresourceNamePrefix, discReq.Node.Id, err)
```

Comment:
not sure how this passed CI. Can you run `BUILD_WITH_CONTAINER=1 make format`, or next time someone touches this file these indent changes will be reverted

Reply:
I did run make format-go - and it changed 1000s of files. This is using golang built-in format.

I'm really not sure why standard go fmt is not good enough here... Buildwithcontainer didn't work last time I tried either.